scripts/globals/items/joyous_serinette.lua

```diff
@@ -0,0 +1,15 @@
+-----------------------------------------
+-- ID: 5425
+-- Item: Joyous Serinette
+-- Item Effect: Change Music
+-----------------------------------------
+require("scripts/globals/msg")
+
+function onItemCheck(target)
+    return 0
+end
+
+function onItemUse(target)
+        target:ChangeMusic(0,214)   --  Eternal Oath
```

Comment:
I'm not sure this is how retail accomplishes this

Reply:
do we have a packet cap of this item being used?

Reply:
ps your indent is wonky

Reply:
@TeoTwawki  fixed the indentention. Only thing it does on reail is just change the music as AoE.
No Dialogs saying that the music was changed.

Fix indentation. Let me know if there's an issue with the way this item is and I can just take it out from the PR and leave the other ones.

Reply:
@TeoTwawki  changed the animation to the correct one, shared the capture for both items being used on discord 😸